turbot/table_turbot_control.go

```diff
@@ -30,6 +30,7 @@ func tableTurbotControl(ctx context.Context) *plugin.Table {
 		Columns: []*plugin.Column{
 			// Top columns
 			{Name: "id", Type: proto.ColumnType_INT, Transform: transform.FromField("Turbot.ID"), Description: "Unique identifier of the control."},
+			{Name: "controls_count", Type: proto.ColumnType_INT, Transform: transform.FromField("Count"), Description: "Total number of controls matching the searched filter in query. It will be same for all the records in the table for a query."},
```

Comment:
Can you please remove this column? I think we'll add the ability to get counts but probably in a separate table.